pkg/jaeger/query/query.go

```diff
@@ -41,47 +45,98 @@ func (p *Query) SpanWriter() spanstore.Writer {
 }
 
 func (p *Query) GetTrace(ctx context.Context, traceID model.TraceID) (*model.Trace, error) {
+	code := "2xx"
```

Comment:
Let's start with `5xx` and set `2xx` when we are 100% sure request is ok. This way if anything breaks along the way we are already in error state.

Reply:
As in https://github.com/timescale/promscale/pull/1102#discussion_r800520731